docs/content/tutorial/0-set-up-your-development-environment.mdx

```diff
@@ -207,6 +207,25 @@ For other distributions, you can check out [PostgreSQL’s website for more guid
 
 You can download PostgreSQL on your macOS using [the installer on their website](https://www.postgresql.org/download/macosx/).
 
+</TabItem>
+<TabItem value="docker" label="Docker">
+
+If you have Docker installed, you can quickly spin up a PostgreSQL instance with the following command:
```

Comment:
After testing this command locally on macOS, I faced an error but it was because the Docker desktop app wasn't running. Let's change the text to the following:

```suggestion
Make sure the Docker Desktop app is running, then run the following command to quickly spin up a PostgreSQL instance:
```

Since the reader chose the Docker tab, it is assumed they already use docker and have it installed